src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example3.java

```diff
@@ -0,0 +1,44 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength">
+      <property name="tokens" value="METHOD_DEF"/>
+      <property name="max" value="4"/>
+      <property name="countEmpty" value="false"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+// xdoc section -- start
+public class Example3 {
+
+  public Example3()  {
+    int var1 = 2;
+    int var2 = 4;
+    int sum = var1 + var2;
+  } // line 5, OK, constructor is not mentioned in the tokens
+
+  public void firstMethod() {
+    // comment - not counted as line
+    System.out.println("line 2");
+  } // line 3, OK, as it allows at most 4 lines
+
```

Comment:
this will result in a violation because they are five lines

Reply:
also the print statement is so long,  will result in a line length violation. I can split it into two lines but in this case, this function will be a bad example (has a length violation will be 6 lines) not a good one so should I make it a bad example?

Reply:
My thought was to make an example that would normally be a violation when we count empty lines to really show this property in action.

Reply:
```
public void secondMethod() {

    // countEmpty property is false,so this line and the line above don't count
    System.out.print("counted as line 2");
  }

```
this example proves this in Example3 If we count empty lines it will be 5 lines so violation

Reply:
Good call.